tests/input-files-ref/test_cubit_tutorial_cubitpy.dat

```diff
@@ -0,0 +1,3169 @@
+
```

Comment:
This test file is quire large. You can pass an optional parameter to the tutorial `cli` function that controls the mesh size. By doing so you can simply create a coarser mesh in testing.

Reply:
Modified